openquake/risklib/scientific.py

```diff
@@ -529,12 +530,19 @@ def __call__(self, iml):
         Intensity Measure Level (IML).
         """
         highest_iml = self.imls[-1]
-
-        if self.no_damage_limit and iml < self.no_damage_limit:
-            return 0.
-        # when the intensity measure level is above
-        # the range, we use the highest one
-        return self.interp(highest_iml if iml > highest_iml else iml)
+        try:
+            len(iml)
+        except TypeError:  # is a scalar
+            if self.no_damage_limit and iml < self.no_damage_limit:
+                return 0.
+            return self.interp(highest_iml if iml > highest_iml else iml)
+        else:  # is a sequence
+            iml = numpy.array(iml)
```

Comment:
Might be better to rename the variable to `imls` since now it can be a sequence of intensity levels.